mirgecom/integrators.py

```diff
@@ -45,6 +48,40 @@ def rk4_step(state, t, dt, rhs):
     return state + dt/6*(k1 + 2*k2 + 2*k3 + k4)
 
 
+def lsrk4_step(state, t, dt, rhs):
+    """Take one step using low storage 4th order Runge-Kutta."""
+    # LSERK coefficients from [Hesthaven_2008]_, Section 3.4
+    a = np.array([
+        0.,
+        -567301805773/1357537059087,
+        -2404267990393/2016746695238,
+        -3550918686646/2091501179385,
+        -1275806237668/842570457699])
+
+    b = np.array([
+        1432997174477/9575080441755,
+        5161836677717/13612068292357,
+        1720146321549/2090206949498,
+        3134564353537/4481467310338,
+        2277821191437/14882151754819])
+
+    c = np.array([
+        0.,
+        1432997174477/9575080441755,
+        2526269341429/6820363962896,
+        2006345519317/3224310063776,
+        2802321613138/2924317926251])
```

Comment:
These constants should be moved outside the function, possibly to an underscored module constant. Otherwise, evaluating them needlessly takes time every time the function is called.

Reply:
Please excuse my python ignorance... 
Does that mean doing something like this at top-level?
```python
_lsrk4_a = <stuff>
_lsrk4_b = <stuff>
_lsrk4_c = <stuff>
```

Reply:
Yeah. Though PEP8 will want those capitalized.